static/scss/embedly.scss

```diff
@@ -32,33 +33,32 @@
       flex-direction: row;
     }
 
-
     .thumbnail {
-      display: flex;
-      overflow: hidden;
-      flex-direction: column;
-      max-height: 300px;
       width: 100%;
-      justify-content: center;
+
+      img {
+        max-height: 600px;
+        object-fit: scale-down;
```

Comment:
this might be kind of an edge case, but with a link-type post it kind of does some strange stuff if the image is small on the mobile view:

![fasfdasdfasdf](https://user-images.githubusercontent.com/6207644/40203881-b27aab3a-59f4-11e8-907a-548c56d8936c.png)